ssi-script.md

```diff
@@ -25,99 +27,77 @@ Specifically, I'd like to find new contributors with different skillsets so we c
-
-#### Slide 8
-
 So what's this Binder project I'm so excited about?
 
-#### Slide 9
+#### Slide 5
 
 mybinder.org is a free-to-use website that allows a code repository to be run in the cloud and easily shared via a single, clickable link.
 It achieves this by capturing a documented software environment from a public repository.
 
-#### Slide 10
+#### Slide 6
 
 BinderHub is the computational infrastructure that underpins mybinder.org and collects together different tools, including JupyterHub and Kubernetes.
 
-#### Slide 11
+#### Slide 7
 
 mybinder.org is a federated project, meaning that it is not hosted by a single team.
 The original Hub is hosted on Google Cloud in the US and a second Hub is hosted on OVH in Europe.
 
-#### Slide 12
+#### Slide 8
 
 The Turing also has a plan to join the federation and host a Hub on Azure.
 Stay tuned!
 
-#### Slide 13
+#### Slide 9
 
-How have I contributed so far?
+How have I contributed to Binder so far?
 
 Through _The Turing Way_, I ran workshops that helped people make Binders out of their own projects.
 I have also helped more people than ever deploy their own BinderHubs and developed a one-click deploy button.
 
-At this year's RSE Conference, I was invited to speak about collaborating with the Binder team on the RSE Worldwide panel, as well as running my BinderHub workshop.
+At this year's RSE Conference, I was invited to speak about collaborating with the Binder team on the RSE Worldwide panel, as well as running my "Build A BinderHub" workshop.
 
-#### Slide 14
+#### Slide 10
 
 What challenges do the Binder community face?
 
 The development team are very Python-centric which makes it tricky to help users with other languages or develop Binder into that space.
 
 The team is also quite small with few people knowing how to maintain the project, which can slow down development.
 
-#### Slide 15
-
-Why should someone be interested in joining the Binder community?
+So why should someone be interested in joining the Binder community?
 
 They would have a voice in shaping the roadmap of the project and also help improve support for their language or domain.
 
 Not forgetting the opportunity to work with wonderful, supportive people!
 
-#### Slide 16
+#### Slide 11
 
-For the SSI Fellowship, I have identified 4 conferences I would like to attend as a Binder advocate.
-The Fellowship should cover the cost of at least three.
+Through the SSI Fellowship I'd like to host a Binder "Do-athon".
+This would be a single day workshop split into two sessions.
+In the morning session, I'd teach the attendees how to use Binder for a their own projects.
+Then in the afternoon, I'd invite them to make a small contribution back to Project Binder around an aspect they may have found tricky.
 
-These are:
+I'm calling this event a "do-athon" as a I want to avoid the association that contributions are only code that, unfortunately, comes with the term "hackathon".
```

Comment:
I would flip this to a positive. 

I'm calling this event a "do-athon" since it encourages contributions of all types, such as code, docs, design, and training. Like Nike says, "Just do it." 😄